__std__/flake.nix

```diff
@@ -16,18 +16,18 @@
   # { inputs, cell }: inputs.nixpkgs, inputs.haskell-nix
   inputs = {
     nixpkgs = {
-      url = "github:NixOS/nixpkgs/34e4df55664c24df350f59adba8c7a042dece61e";
+      follows = "haskell-nix/nixpkgs";
```

Comment:
This is a major change: we're deliberately not tying ourselves to the `haskell.nix` nixpkgs at the moment. I'd rather keep it that way unless we have a good reason not to.

Reply:
And it forces you to fix the R thing!